libraries/SPI/src/SPI.cpp

```diff
@@ -421,3 +421,41 @@ void SPIClass::detachInterrupt(void)
 {
   // Should be disableInterrupt()
 }
+
+#if defined(SUBGHZSPI_BASE)
+SUBGHZSPIClass SubGHZ_SPI;
+
+void SUBGHZSPIClass::begin(uint8_t _pin)
+{
+  if (_pin != CS_PIN_CONTROLLED_BY_USER) {
+    LL_PWR_SelectSUBGHZSPI_NSS();
```

Comment:
Why select this pin here? The original begin(pin) implementation just configures the pin to output high (unselected), so I guess only an unselect is needed here?

Reply:
Also, the superclass actually does a `pinMode` on the passed `_pin`, I'm not sure if this is problematic? Or actully, this is probably handled because `_spi.pin_ssel` is always set by the `SUBGHZSPIClass` constructor.

Also, shouldn't this code (and also below) actually check *which* pin is passed? Now it just modifies the internal NSS pin regardless of what was passed, which might be surprising (which could show up in the theoretical usecase of using this SPI bus in debug mode to do write-only SPI transactions to an externally connected device).

Reply:
Right for the first point.
About the second, now we call all SPIClass with CS_PIN_CONTROLLED_BY_USER .